src/random.c

```diff
@@ -11,7 +11,12 @@
 
 #include <stdint.h>
 
-int fido_get_random(void *buf, size_t len) {
+#if defined(NO_SOFTWARE_RNG)
```

Comment:
In other parts of the code ifdef is used

Reply:
Kinda mixed, yeah. At least for the other `NO_` makros we used `if defined`.

Reply:
ok doesn't really matter I guess